src/js/components/Widgets/CopyLinkModal.jsx

```diff
@@ -44,7 +44,7 @@ export default class CopyLinkModal extends Component {
     }
 
     return (
-      <Modal {...this.props} size="large" aria-labelledby="contained-modal-title-lg">
+      <Modal Cosize="large" aria-labelledby="contained-modal-title-lg">
```

Comment:
Hello @Julian0510, thank you for submitting this pull request. On this line, 'size="large"' needs to remain as is.  The text "Cosize" looks like a typo? The letters "Co" should be removed.

I believe we want the show and onHide props passed into "<Modal" (as opposed to passing into the "<input" around line 53.